From 33015546c62a7b73793d62983a84b097362cec1a Mon Sep 17 00:00:00 2001
From: Jan Beich <jbeich@FreeBSD.org>
Date: Mon, 19 Aug 2024 12:46:36 +0000
Subject: [PATCH 09/20] config: add missing header for libc++ after
 92744b5b9aa3 (#7403)

In file included from src/pch/pch.hpp:1:
In file included from src/Compositor.hpp:11:
src/config/ConfigManager.hpp:147:10: error: no template named 'variant' in namespace 'std'
  147 |     std::variant<SBoolData, SRangeData, SFloatData, SStringData, SColorData, SChoiceData, SGradientData, SVectorData> data;
      |     ~~~~~^
---
 src/config/ConfigManager.hpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/config/ConfigManager.hpp b/src/config/ConfigManager.hpp
index 4241031b..4d087753 100644
--- a/src/config/ConfigManager.hpp
+++ b/src/config/ConfigManager.hpp
@@ -6,6 +6,7 @@
 #include "../debug/Log.hpp"
 #include <unordered_map>
 #include "../defines.hpp"
+#include <variant>
 #include <vector>
 #include <deque>
 #include <algorithm>
-- 
2.45.2

